ids/test_generator.go

```diff
@@ -19,6 +19,15 @@ func GenerateTestShortID() ShortID {
 	return newShortID
 }
 
+// BuildTestShortID is an utility to build ShortID from bytes in UTs
+// It must not be used in production code. In production code we should
+// use ToShortID, which performs proper length checking.
+func BuildTestShortID(src []byte) ShortID {
+	res := ShortID{}
+	copy(res[:], src)
+	return res
+}
+
```

Comment:
Why are we adding this function? Can't we just directly specify the `ShortID`?

Reply:
Dropped